### DISCUSSION TITLE
Future ideas

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/Perl-1.0#33] [Q&A] [answered]
What will you do about Perl 1 in the future?

**Comment by Stagyrite** [comment, marked as answer]
Coding Perl 1 seems to be a kicking dead horse. Notwithstanding, the compilers should be fresh, and Perl 1 needs more coding. There are two other Perl repositories ([GitHub/eltikia/perl1](https://github.com/eltikia/perl1) and [GitHub/eltikia/perl-1.0_16](https://github.com/eltikia/perl-1.0_16)). There might be some effort needed to make them compile. As for the [GitHub/eltikia/Perl-1.0](https://github.com/eltikia/Perl-1.0) repository, compiler warnings will be removed.
